### PR TITLE
Expose video options through the config object

### PR DESCRIPTION
**Pull request by axelboberg**
- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Feature

- **What is the current behavior?** (You can also link to an open issue here)
Video options are not exposed.

- **What is the new behavior (if this is a feature change)?**
Video options are now exposed through the config object, allowing for setting `videoBitrate`, `videoCodec`, `videoCrf`, `videoPreset` and `videoPixelFormat` to custom values.

### CODE REVIEW

File: src/lib/pageVideoStreamWriter.ts [hunk 1]
```diff
@@ -172,16 +172,17 @@ export default class PageVideoStreamWriter extends EventEmitter {
...
       .size(this.videoFrameSize)
       .aspect(this.options.aspectRatio || '4:3')
       .autopad(this.autopad.activation, this.autopad?.color)
       .inputFormat('image2pipe')
       .inputFPS(this.options.fps)
-      .outputOptions('-preset ultrafast')
-      .outputOptions('-pix_fmt yuv420p')
-      .outputOptions('-minrate 1000')
-      .outputOptions('-maxrate 1000')
+      .outputOptions(`-crf ${this.options.videoCrf ?? 23}`)
```
Comment (prasanaworld) [inline src/lib/pageVideoStreamWriter.ts:181]: Could you please add a bit of description stating why it is defaults to 23 ?

**Comment by prasanaworld** [comment]
Looks good otherwise. Thanks

File: src/lib/pageVideoStreamWriter.ts (hunk 1 above)
Comment (jrabek) [reply src/lib/pageVideoStreamWriter.ts:181]: Could quote this page? https://trac.ffmpeg.org/wiki/Encode/H.264  "23 is the default"

**Comment by prasanaworld** [comment]
Hey @axelboberg, Could you please resolve conflict and update the PR  or let me know I can do it and merge the PR

**Comment by axelboberg** [comment]
Done - although I'm leaving the `-framerate 1` option in place that has been added after the PR was created. It looks strange to me and there isn't a ton of documentation of it, but maybe someone can motivate it.

**Pull request merged**